### PR TITLE
upsert for ErrorTypes

### PR DESCRIPTION
**Pull request by eacharles**

### CODE REVIEW

File: src/lsst/cm/tools/db/sqlalch_interface.py [hunk 1]
```diff
@@ -586,6 +586,26 @@ def load_error_types(self, config_yaml: str) -> None:
         error_code_dict = config_data["pandaErrorCode"]
         for key, val in error_code_dict.items():
             for error_name, error_type in val.items():
+                matched_id = conn.query(ErrorType.id).filter_by(error_name=error_name).first()
+                if matched_id is not None:
+                    stmt = (
+                        update(ErrorType)
+                        .where(ErrorType.id == matched_id[0])
```
Comment (fritzm) [inline src/lsst/cm/tools/db/sqlalch_interface.py:593]: Do you need/want `[0]` here, given `first()` above?

Comment (eacharles) [reply src/lsst/cm/tools/db/sqlalch_interface.py:593]: yes I do

Comment (eacharles) [reply src/lsst/cm/tools/db/sqlalch_interface.py:593]: It don't work otherwise

Comment (fritzm) [reply src/lsst/cm/tools/db/sqlalch_interface.py:593]: Well alrighty then!  It doesn't do what I thought it did, I guess -- will have to educate myself later.

**Review by fritzm — APPROVED** [review]

**Comment by eacharles** [comment]
It returns a TupperwareOn Jun 2, 2023, at 8:13 PM, Fritz Mueller ***@***.***> wrote:﻿
@fritzm approved this pull request.

**Comment by eacharles** [comment]
Tuple On Jun 2, 2023, at 8:33 PM, Eric Charles ***@***.***> wrote:﻿It returns a TupperwareOn Jun 2, 2023, at 8:13 PM, Fritz Mueller ***@***.***> wrote:﻿
@fritzm approved this pull request.

**Pull request merged**